docs/source/dev_guide/testing.rst

```diff
@@ -1,22 +1,60 @@
 Testing E3SM Diagnostics
 ==============================================
 
-Unit tests
-----------
+Unit and integration tests
+--------------------------
 
-Run all unit tests by doing the following:
+Run all automated tests by doing the following:
 
     .. code::
 
         pip install . # Install your changes
-        ./tests/test.sh # Run all unit tests
+        ./tests/test.sh # Run all unit and integration tests
+
+
+If these tests pass, you're done. If they fail unexpectedly however,
+your code might have changed what the output looks like.
+
+``ls tests/system/image_check_failures`` will show your all the images that differ from expected.
+
+If you see unexpected images listed, you'll have to investigate why your
+code changed what the images look like.
+
+If the only images listed are ones you expected to see changed,
+then you need to update the expected images.
+
+
+    .. code::
+
+        cd /lcrc/group/e3sm/public_html/e3sm_diags_test_data/integration/expected
+        cat README.md
+        # This will show you the version, date, and hash of the current expected images.
```

Comment:
Perhaps I should note to do this step AFTER merging, so that `<hash>` will match the hash of the squashed merge commit.